src/frontend/shortlist/src/views/LoginView.vue

```diff
@@ -0,0 +1,14 @@
+<script>
```

Comment:
It is a redundant file as there is a Demo file that can handle login and signup both. Should we remove it?

Reply:
This is not a redundant file. Do not remove. I may change the name later.